wrapper.go

```diff
@@ -238,6 +242,10 @@ func New(e *echo.Echo, docPath string, i *Info) ApiRoot {
 	return r
 }
 
+func (r *Root) ADD(method, path string, h echo.HandlerFunc, m ...echo.MiddlewareFunc) Api {
```

Comment:
Can you please change the method name to `Add()`?
I think it's better to keep the same name as Echo's.